### ISSUE TITLE
Native

### ISSUE THREAD

**Issue by mgriese** [obophenotype/human-phenotype-ontology#6679]
**Preferred term label:**
Native

**Synonyms**

**Definition (free text, please give PubMed ID)**
Native (belonging to the lung by origin, manifesting over the life time, i.e. before, at, after birth during childhood, adolescence, adulthood or senescence) parenchymal disorders (develop over lifetime). Native means here during the biography, during development until death. The condition may be inherited, but manifestation may be over the lifetime.
Not synonymous to congenital or hereditary, which indicate that the condition was acquired before birth.
--
Usage of the term: PMID: 1990717, PMID: 16282461

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
Can we add a modifier underneath ``Temporal pattern HP:0011008``

https://hpo.jax.org/app/browse/term/HP:0011008

**Comment by pnrobinson** [comment]
After some thought I do not think this is in scope for HPO since this modifier encompasses a good deal of clinical intuition, termporality, and is also specific to pulmonology. Possibly this could be a category in Mondo, but I think it is probably something that might be hard to capture well with ontology analysis.
